Tools/CMake/AMReXThirdPartyLibraries.cmake

```diff
@@ -73,3 +73,19 @@ if (AMReX_PETSC)
     find_package(PETSc 2.13 REQUIRED)
     target_link_libraries( amrex PUBLIC PETSC )
 endif ()
+
+#
+# SUNDIALS
+#
+if (AMReX_SUNDIALS)
+    find_package(SUNDIALS 5.7.0 REQUIRED)
```

Comment:
This causes a failure in the Nyx superbuild: 
https://github.com/AMReX-Astro/Nyx/runs/2513476050?check_suite_focus=true#step:4:102

If I use an installed version of sundials, I get an error from some of the components (such as IDA not being installed and failing an alias setup in SUNDIALSConfig.cmake

If I comment this line out I can generate and build Nyx, but get a runtime error for Cuda during init.